deployments.md

```diff
@@ -202,6 +202,14 @@ in any language, and within any legislation</p>
                         <p class="deployment__link"><a href="https://chiedi.dirittodisapere.it/">https://chiedi.dirittodisapere.it/</a></p>
                     </div>
                 </div>
+                --><div class="deployments__unit--minor">
```

Comment:
Looks like this `-->` may be a stray closing comment.

There should either be an opening comment (`<!--`) at the end of the previous line, or none at all.